scripts/dev/newReleaseBranch.py

```diff
@@ -0,0 +1,549 @@
+#######################################
+# This script creates HotFix branches for repo, share and packaging projects, and updates the master branches ready for the next SP/major release.
+# To use this script you need to install lxml:
+# 'pip install lxml'
+# Run the script without passing any arguments or with -h/--help argument to get usage information.
+# See below script behaviour explained.
+#######################################
+# Create a HotFix branches for the released version (for X.Y.Z release it will be X.Y.N eg., create 23.2.N for 23.2.0 release)
+# 1. acs-packaging:
+# - set RELEASE_VERSION to X.Y.Z+1, DEVELOPMENT_VERSION to X.Y.Z+1-SNAPSHOT in master_release.yml
+# - set POM versions to X.Y.Z+1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# 2. enterprise-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y.Z+1
+# - set POM versions to  X.Y.Z+1.1-SNAPSHOT
+# 3. enterprise-repo:
+# - set POM versions to  X.Y.Z+1.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - set acs.version.label to .1
+# 4. community-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y.Z+1
+# - set main POM versions to  X.Y.Z+1.1-SNAPSHOT
+# 5. community-repo:
+# - set ACS version properties in main POM to X.Y.Z+1
+# - set POM versions to  X.Y.Z+1.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - increment schema by 1 in repository.properties?
+# - set version.revision to Z+1 in version.properties (test resources)
+# 6. acs-community-packaging
+# - set RELEASE_VERSION to X.Y.Z+1, DEVELOPMENT_VERSION to X.Y.Z+1-SNAPSHOT in ci.yml
+# - set POM versions to X.Y.Z+1-SNAPSHOT?
```

Comment:
We only do HF releases for Enterprise, so I think the last digit will always be `0` for acs-community-packaging.

Reply:
I didn't know that either (HF for ent only). If that is the case, then we could possibly even abandon HF branch creation. WDYT?

Reply:
I don't think we should need HF branches in acs-community-packaging, but I just checked and they all seem to be there. Hardly any HF releases are there though. I suggest we get rid of them and add them back if we hit a problem.

Reply:
Excluded comm-packaging from HF branch creation.